src/charm.py

```diff
@@ -300,20 +340,19 @@ def _on_update_status(self, _) -> None:
                 self.peers.data[unit].get("member-state", "unknown") for unit in self.peers.units
             }
 
-            if all_states == {"offline"} and self.unit.is_leader():
-                # All instance are off, reboot cluster from outage from the leader unit
-
+            if all_states in [{"offline"}, set()] and self.unit.is_leader():
+                # All instance are off or its a single unit cluster
+                # reboot cluster from outage from the leader unit
                 logger.debug("Attempting reboot from complete outage.")
                 try:
-                    self._mysql.reboot_from_complete_outage()
+                    # reboot from outage forcing it when it a single unit
+                    self._mysql.reboot_from_complete_outage(force=all_states == set())
```

Comment:
it may be easier to understand if `force=len(self.peers.units == 0)`

Reply:
this is going out, since we should not never `force`